authMiddleware.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require("jsonwebtoken");
+
+module.exports = async (req, res, next) => {
+    let token;
+    if (
+        req.headers.authorization &&
+        req.headers.authorization?.includes("Bearer")
+    ) {
+        try {
+            token = req.headers.authorization.split(" ")[1];
+            const user = jwt.verify(token, "Secretkey123");
+            req.body.user = user;
+            next();
+        } catch (error) {
+            // TODO check if TokenExpiredError
```

Comment:
you probably should refresh your token here

Reply:
The refreshToken functionality is implemented on the frontend. We do not pass the refreshToken for the getProject request.